### PR TITLE
adding the files of the ticket  16

### PR DESCRIPTION
**Pull request by AlexDanielGutierrezCardenas**
files [WEBSITE-16]

### CODE REVIEW

**Comment by olmedo90** [comment]
image

It is not according to Figma. fix the following: (check the size of the components, positioning of layers and the color of the texts)

**Comment by olmedo90** [comment]
image
 
It is according to the figma. Continue with the same interest.👍

File: src/ui/components/HistorialPedidos/FiltrarPedidos.jsx [hunk 1]
```diff
@@ -12,13 +12,8 @@ import {SvgIcon} from '@material-ui/core';
 import { addDays,format } from 'date-fns';
 import { useState,useEffect } from 'react';
 import { Calendar, DateRange } from 'react-date-range'
-<<<<<<< HEAD
 // 
 
-=======
```
Comment (Ditmar) [inline src/ui/components/HistorialPedidos/FiltrarPedidos.jsx]: remove you have conflicts in the pr

File: src/ui/components/HistorialPedidos/FiltrarPedidos.jsx [hunk 2]
```diff
@@ -44,31 +39,10 @@ const handleChanges = (event, newValue) => {
...
-      [event.target.name]: event.target.checked,
-    });
-  };
-
-  const { gilad, jason, antoine ,alex,roma} = state;
-  const IconLinear=(props)=>{
-    return (
-      <SvgIcon {...props}>
-        <path d="M0.714355 0.857147L5.00007 5.14286L9.28578 0.857147" stroke="#54595E" stroke-linecap="round" stroke-linejoin="round"/>
->>>>>>> 4f158bd19a70584a1352ff6855ae646539b582e0
```
Comment (Ditmar) [inline src/ui/components/HistorialPedidos/FiltrarPedidos.jsx]: more conflicts

File: src/ui/components/HistorialPedidos/FiltrarPedidos.jsx [hunk 3]
```diff
@@ -99,15 +72,12 @@ const handleChanges = (event, newValue) => {
   console.log('cerrado');
   setIsOpen(false);
  }
-=======
->>>>>>> 4f158bd19a70584a1352ff6855ae646539b582e0
```
Comment (Ditmar) [inline src/ui/components/HistorialPedidos/FiltrarPedidos.jsx]: more conflicts

**Review by Ditmar — CHANGES_REQUESTED** [review]
many conflicts  in the pr

**Review by Ditmar — APPROVED** [review]
nice work approved

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**